api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -191,7 +191,7 @@ def validate_booking_by_token(token: str) -> None:
     except exceptions.BookingIsNotConfirmed as exc:
         raise api_errors.ForbiddenError({"booking": str(exc)})
 
-    bookings_api.mark_as_used(booking)
+    bookings_api.mark_as_used(booking, author_set_used=booking_models.BookingAuthorType.OFFERER)
```

Comment:
```suggestion
    bookings_api.mark_as_used(booking, booking_models.BookingAuthorType.OFFERER)
```
Il me semble que préciser le nom de la variable n'est pas nécessaire

Reply:
(il faut l'enlever partout si elle ne l'est effectivement pas)